pkgs/development/python-modules/openmm/default.nix

```diff
@@ -0,0 +1,87 @@
+{ buildPythonPackage, lib, fetchFromGitHub, cmake, clang, gfortran, fftwSinglePrec
+, doxygen, ocl-icd, opencl-headers, swig, python, cython, numpy
+, cudatoolkit, enableCuda ? false
+} :
```

Comment:
```suggestion
{ buildPythonPackage
, lib
, fetchFromGitHub
, cmake
, clang
, gfortran
, fftwSinglePrec
, doxygen
, ocl-icd
, opencl-headers
, swig
, python
, cython
, numpy
, enableCuda ? false, cudatoolkit
}:
```

Reply:
Hm why? :slightly_smiling_face:  nixfmt agrees with this format and many other packages use it.

Reply:
Well, other packages also use this format, plus it's easier to manage diffs with something like this. Speaking of formatting, AFIAK there are no standard formatters for nixpkgs. There's a `nixpkgs-fmt` tho, which I guess is more geared towards nixpkgs.

Reply:
You are right about the diffs. The nix tooling gets me every time ... However, has the suggested format now as by nixpkgs-fmt